res/controllers/Denon-Prime-4-scripts.js

```diff
@@ -168,14 +168,20 @@ const colDeckDark = [
     Prime4.rgbCode[deckColors[3] + "Dark"],
 ];
 
-Prime4.init = function() {
-    midi.sendShortMsg(0x90, 0x75, 0x00); // Turn off all LEDs
+Prime4.init = function(id, debug) {
+    // Turn off all LEDs
+    midi.sendShortMsg(0x90, 0x75, 0x00);
+
+    // Initialise all three physical sections of the unit
     Prime4.leftDeck = new Prime4.Deck([1, 3], 4);
     Prime4.rightDeck = new Prime4.Deck([2, 4], 5);
     Prime4.mixerStrip = new components.ComponentContainer();
+
+    // Register '0x9n' as a button press and 'ox8n' as a button release
     components.Button.prototype.isPress = function(channel, control, value, status) {
         return (status & 0xF0) === 0x90;
     };
```

Comment:
Typo, also I'd put the overwrites outside the `init` function somewhere else at the top of the mapping. Otherwise we could've already received midi messages before changing the `isPress` behavior. 
```suggestion
    // Register '0x9n' as a button press and '0x8n' as a button release
    components.Button.prototype.isPress = function(channel, control, value, status) {
        return (status & 0xF0) === 0x90;
    };
```

Reply:
Thank you, I'll fix the typo soon. The documentation suggested putting this section inside the .init function, but I can move it 👍

Reply:
Can you link me to the exact location? I'd like to look into that.

Reply:
It's here: https://github.com/mixxxdj/mixxx/wiki/Components%20JS#button
Right before the PlayButton section

Reply:
Thank you. I have no idea why its suggested like this. I simply removed it for now.